Python/Product/ExternalProfilerDriver/VTuneToDWJSON.cs

```diff
@@ -94,11 +94,11 @@ class VTuneToDWJSON {
         /// <paramref name="filename"/>
         /// </summary>
         /// <param name="filename">The filename with the callstack report</param>
-        public static double CSReportToDWJson(string filename, string outfname) {
+        public static double CSReportToDWJson(string filename, string outfname, string sympath = "") {
```

Comment:
the names of the params could be more specific, especially the first one.

the 3rd param is called `symbolPath` in `AddLineNumbers`, that would be better than `sympath`